docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+# Introduction to ERC721 (NFT) Smart Contracts
+
+This tutorial will start you with a basic ERC721 (NFT) smart contract on the Avalanche Network, regardless of your previous development experience. We will deploy our NFT on the Avalanche Fuji Testnet and view it on the Snowtrace Testnet Explorer. Note that these are not transferable to the Mainnet. However, once you feel comfortable launching your project, you can do so on Avalanche Mainnet and list it on an NFT marketplace.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "We'll deploy our NFT..." instead of "We will deploy our NFT..."